app/vmauth/auth_config.go

```diff
@@ -149,6 +150,15 @@ func (h *Header) MarshalYAML() (interface{}, error) {
 	return h.sOriginal, nil
 }
 
+func shouldDropHostHeader(headers []*Header) bool {
```

Comment:
Naming looks a bit confusing to me. I'd expect from `dropHostHeader` option to completely remove host header from request and leave it empty.

 Maybe it's better to use - overrideHostHeader or enforceHostHeader WDYT?

Reply:
updated